system/Security/CheckPhpIni.php

```diff
@@ -118,23 +118,30 @@ private static function outputForWeb(array $output, array $thead, array $tbody):
-            'display_errors'          => ['recommended' => '0'],
-            'display_startup_errors'  => ['recommended' => '0'],
-            'log_errors'              => [],
-            'error_log'               => [],
-            'default_charset'         => ['recommended' => 'UTF-8'],
-            'memory_limit'            => ['remark' => '> post_max_size'],
-            'post_max_size'           => ['remark' => '> upload_max_filesize'],
-            'upload_max_filesize'     => ['remark' => '< post_max_size'],
-            'request_order'           => ['recommended' => 'GP'],
-            'variables_order'         => ['recommended' => 'GPCS'],
-            'date.timezone'           => ['recommended' => 'UTC'],
-            'mbstring.language'       => ['recommended' => 'neutral'],
-            'opcache.enable'          => ['recommended' => '1'],
-            'opcache.enable_cli'      => [],
-            'opcache.jit'             => [],
-            'opcache.jit_buffer_size' => [],
+            'error_reporting'                 => ['recommended' => '5111'],
+            'display_errors'                  => ['recommended' => '0'],
+            'display_startup_errors'          => ['recommended' => '0'],
+            'log_errors'                      => [],
+            'error_log'                       => [],
+            'default_charset'                 => ['recommended' => 'UTF-8'],
+            'memory_limit'                    => ['remark' => '> post_max_size'],
+            'post_max_size'                   => ['remark' => '> upload_max_filesize'],
+            'upload_max_filesize'             => ['remark' => '< post_max_size'],
+            'request_order'                   => ['recommended' => 'GP'],
+            'variables_order'                 => ['recommended' => 'GPCS'],
+            'date.timezone'                   => ['recommended' => 'UTC'],
+            'mbstring.language'               => ['recommended' => 'neutral'],
+            'opcache.enable'                  => ['recommended' => '1'],
+            'opcache.enable_cli'              => [],
+            'opcache.jit'                     => [],
+            'opcache.jit_buffer_size'         => [],
+            'opcache.memory_consumption'      => ['recommended' => '728', 'remark' => 'Increasing the configured memory size (MB) will improve performance by caching those files (consideration based on free space of memory)'],
+            'opcache.interned_strings_buffer' => ['recommended' => '64'],
+            'opcache.max_accelerated_files'   => ['recommended' => '56000', 'remark' => 'Increasing the configured accelerated will many cache files into OPCache hash table'],
+            'opcache.max_wasted_percentage'   => ['recommended' => '15'],
+            'opcache.validate_timestamps'     => ['recommended' => '0'],
+            'opcache.revalidate_freq'         => ['recommended' => '0'],
+            'opcache.save_comments'           => ['recommended' => '1'],
```

Comment:
Can you show the reasons why these values are recommended for most production environments?

Reply:
In `memory_consumption` and `max_accelerated_files` is many server have i think (RAM 1GB minimum) and i remark you can adjustment to your server, will give your improve to CACHING files project without effecting by changing coding before you restart PHP Engine (especially `validate_timestamps` with `0`).

Reply:
Most likely "more" is always better, but what are the recommendations based on? Any article or your experience? 
Right now someone can say we should double these recommendations because it will be better for performance.

Reply:
> Most likely "more" is always better

No, the memory is limited. So too much "more" may be worse. If one PHP process takes huge memory, we cannot have many processes.

The following is an answer by Google Gemini.

---
There are recommended values for these PHP ini settings in production environments, but finding the absolute perfect settings depends on your specific application and server workload. Here's a breakdown of the settings with recommendations and why they matter:

**1. opcache.memory_consumption:**

* **Recommendation:**  128M - 512M. This is a good starting point for most servers.
* **Why it matters:** This setting allocates memory for the Opcache engine to store precompiled PHP scripts. Higher values improve performance by reducing script load times, but it also consumes server RAM.

**2. opcache.interned_strings_buffer:**

* **Recommendation:**  4M - 8M. Adjust based on your application's string usage.
* **Why it matters:** This allocates memory for frequently used strings within PHP code. A well-tuned value can improve performance, but excessively high values might waste RAM.

**3. opcache.max_accelerated_files:**

* **Recommendation:**  1024 - 4096. Can be higher for larger applications.
* **Why it matters:** This limits the number of PHP scripts Opcache can store. Setting it too low might not cache all your scripts, while too high can use excessive memory.

**4. opcache.max_wasted_percentage:**

* **Recommendation:**  5 - 15. 
* **Why it matters:** This defines the percentage of wasted memory in the Opcache cache before it triggers a cleanup. A higher value allows more scripts to be cached, but wasted memory reduces efficiency.

**5. opcache.validate_timestamps:**

* **Recommendation:**  On (True).
* **Why it matters:** Enabling this ensures Opcache checks script timestamps for updates before serving them. Disabling might lead to outdated code being served.

**6. opcache.revalidate_freq:**

* **Recommendation:**  180 (seconds). Adjust based on your deployment process.
* **Why it matters:** This defines how often Opcache checks for script modifications. A lower value ensures faster updates but uses more resources.  

**7. opcache.save_comments:**

* **Recommendation:**  Off (False) in production.
* **Why it matters:** Enabling this stores comments within the cached scripts, increasing cache size but offering no benefit in production.

**Remember:** These are starting points. It's recommended to monitor your server performance and Opcache stats to fine-tune these settings for your specific environment.  There are also tools like [PHP opcache status](URL php opcache status ON php.net) to help analyze Opcache performance.

Reply:
I agree `memory_consumption` with 728MB too much, Btw i make this by case in my production server. No need `validate_timestamps` with `ON` and `revalidate_freq` with 1-xx seconds, restart FPM in the next deploy.

For too wide, need to change description in remark.

Reference:
https://gist.github.com/rohankhudedev/1a9c0a3c7fb375f295f9fc11aeb116fe
https://tideways.com/profiler/blog/fine-tune-your-opcache-configuration-to-avoid-caching-suprises

Reply:
> No, the memory is limited. So too much "more" may be worse. If one PHP process takes huge memory, we cannot have many processes.

Yes, I meant that the values will vary depending on the available resources.